### ISSUE TITLE
AppImage auto-update

### ISSUE THREAD

**Issue by RafaelLinux** [Heroic-Games-Launcher/HeroicGamesLauncher#3561]
### Feature description

I love HL and I think it's hard to improve it, but you always achieve something more, thank you.

As a user of the AppImage format (for me, the best way to manage applications) I would appreciate it if you could publish the self-updating AppImage, like other applications that use this format.

**Comment by flavioislima** [comment]
Well, we have auto update for AppImage for at least a couple of years.
If it is not working it's because we broke it at some point.

Can you download the appimage from the previous version and launch it from the terminal and check if it will show any error related with the update?

**Comment by RafaelLinux** [comment]
I had just the previous version, and it didn't auto-update.

**Comment by duncreg** [comment]
auto-update had been working with AppImage but it is broken for this most recent update.

**Comment by davidfragalaureano** [comment]
Hey, I'm following this issue, but came across this PR:
- https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/pull/4536

Which basically disables auto-update for Linux, that may explain why it doesn't work.

Haven't tested removing those changes to see if auto-update works, wanted to make sure I'm on the right path.

**Comment by RafaelLinux** [comment]
I never tried RPM version. I thought it didn't exist when I begin to use Heroic, so I was referring to AppImage version auto-update.
Anyway, I'll give a chance to RPM version to see if it is better option.

Thanls!!

**Comment by davidfragalaureano** [comment]
@RafaelLinux I've just downloaded a [previous](https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/releases/tag/v2.16.1) release in the `AppImage` format, and as soon as I started it, It gave me the option to update it,

Is that what you are referring to?

<img width="1280" height="852" alt="Image" src="https://github.com/user-attachments/assets/7acb5088-3a29-4df3-b117-dc6415c721bc" />

**Comment by RafaelLinux** [comment]
No, is not that way of updating. AppImage can implement an "autoupdate" . It warns user about a download and if user acccept, is the AppImage who download and replace bin file.

**Comment by arielj** [comment]
I understand this was fixed in one of the latest releases https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/releases/tag/v2.19.1
